assets/js/common/Pagination/Pagination.test.jsx

```diff
@@ -85,4 +85,170 @@ describe('Pagination component', () => {
+    }
+  );
+
+  it.each`
+    currentPage | pages | nextPage
+    ${1}        | ${5}  | ${2}
+    ${2}        | ${5}  | ${3}
+    ${5}        | ${5}  | ${5}
+    ${5}        | ${15} | ${6}
+    ${15}       | ${99} | ${16}
+    ${1}        | ${99} | ${2}
+    ${99}       | ${99} | ${99}
+  `(
+    'should select next page ($pages, $currentPage)',
+    async ({ pages, currentPage, nextPage }) => {
+      const user = userEvent.setup();
+      const onSelect = jest.fn();
+
+      render(
+        <Pagination
+          pages={pages}
+          currentPage={currentPage}
+          currentItemsPerPage={10}
+          itemsPerPageOptions={[10, 20, 50]}
+          onSelect={onSelect}
+          onChangeItemsPerPage={noop}
+        />
+      );
+
+      await act(() => user.click(screen.getByText(`>`)));
+      expect(onSelect).toHaveBeenCalledWith(nextPage);
+    }
+  );
+
+  it('should work correctly as controlled component', async () => {
+    const user = userEvent.setup();
+
+    const onSelect = jest.fn();
+    const initialPage = 1;
+
+    function ControlledComponent() {
+      const [value, setValue] = useState(initialPage);
+      return (
+        <Pagination
+          pages={3}
+          currentPage={value}
+          currentItemsPerPage={10}
+          itemsPerPageOptions={[10, 20, 50]}
+          onSelect={(selected) => {
+            onSelect(selected);
+            setValue(selected);
+          }}
+          onChangeItemsPerPage={noop}
+        />
+      );
+    }
+
+    render(<ControlledComponent />);
+
+    const actions = [
+      { action: `<`, expected: 1 }, // previous of the first is the first
+      { action: `>`, expected: 2 },
+      { action: `>`, expected: 3 },
+      { action: `>`, expected: 3 }, // next of the last is the last
+      { action: `<`, expected: 2 },
+    ];
+
+    for (let i = 0; i < actions.length; i += 1) {
+      // eslint-disable-next-line no-await-in-loop
+      await act(() => user.click(screen.getByText(actions[i].action)));
+      expect(onSelect).toHaveBeenCalledWith(actions[i].expected);
+      expect(onSelect).toHaveBeenCalledTimes(1);
+      onSelect.mockClear();
+    }
+  });
+
+  it.each`
+    currentPage | pages | selected
+    ${1}        | ${5}  | ${1}
+    ${3}        | ${5}  | ${3}
+    ${5}        | ${5}  | ${5}
+    ${6}        | ${5}  | ${5}
+    ${1}        | ${99} | ${1}
+    ${21}       | ${99} | ${21}
+    ${99}       | ${99} | ${99}
+    ${200}      | ${99} | ${99}
+  `(
+    'should highlight the current page ($pages, $currentPage)',
+    ({ pages, currentPage, selected }) => {
+      render(
+        <Pagination
+          pages={pages}
+          currentPage={currentPage}
+          currentItemsPerPage={10}
+          itemsPerPageOptions={[10, 20, 50]}
+          onSelect={noop}
+          onChangeItemsPerPage={noop}
+        />
+      );
+
+      expect(screen.getByText(`${selected}`)).toHaveStyle(
+        'color: rgb(48 186 120)'
+      );
```

Comment:
It's a good point; honestly, I don't have an answer. 

One argument in favor of using the style is that you are testing against what the user sees, which in this case is a green link. If we have to change the color for any reason, I would say it will be no big deal to change the test accordingly.